### PR TITLE
Allow non-administrative read access to expectedbuild.php

### PR DESCRIPTION
**Pull request by zackgalbreath**
Don't require an admin account to see how long a build has been AWOL.

### CODE REVIEW

**Comment by jamiesnape** [comment]
Is this in line with the rest of CDash's permissions model, if one exists?

File: public/api/v1/expectedbuild.php [hunk 1]
```diff
@@ -44,12 +44,7 @@
 $buildname = htmlspecialchars(pdo_real_escape_string($_REQUEST['name']));
 $buildtype = htmlspecialchars(pdo_real_escape_string($_REQUEST['type']));
 
-// Make sure the user has admin access to this project.
-if (!isset($_SESSION['cdash']) || !isset($_SESSION['cdash']['loginid'])) {
-    $response['requirelogin'] = 1;
-    echo json_encode($response);
-    return;
-}
+// Make sure the user has access to this project.
```
Comment (jamiesnape) [inline public/api/v1/expectedbuild.php]: The PR title says "anonymous".

Comment (zackgalbreath) [inline public/api/v1/expectedbuild.php]: Good point, renamed.  We still need this check for private projects, but we don't need to check for administrative rights on GET requests here.

Comment (jamiesnape) [inline public/api/v1/expectedbuild.php]: Makes sense now.

**Comment by jamiesnape** [comment]
LGTM.

**Pull request merged**